alert-policies/dbt-cloud/dbt-job-failed.yml

```diff
@@ -0,0 +1,72 @@
+# Name of the alert
+name: Dbt Job Failed
+
+# Description and details
+description: |+
```

Comment:
This description should be less than 240 characters

Reply:
Updated